src/test/java/tools/jackson/databind/jsontype/PolymorphicDeserErrorHandlingTest.java

```diff
@@ -54,7 +54,7 @@ public void testUnknownClassAsSubtype() throws Exception
         ObjectReader reader = MAPPER.readerFor(BaseUnknownWrapper.class)
                 .without(DeserializationFeature.FAIL_ON_INVALID_SUBTYPE);
         BaseUnknownWrapper w = reader.readValue(a2q
-                ("{'value':{'clazz':'com.foobar.Nothing'}}'"));
```

Comment:
Removing unneccesary `"` double quote. The test is not about unexpected token after JSON.